### PR TITLE
Updates section on the Quality Criteria

### PR DESCRIPTION
**Pull request by andreban**
- Removes Lighthouse 80
- Adds details on Chrome 86 crashes.

@b1tr0t PTAL

### CODE REVIEW

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 1]
```diff
@@ -66,8 +66,23 @@ nightly version.
 ### Quality Criteria
 
 Web developers should use a Trusted Web Activity when they want to include web content in an
-Android app. Web content in a Trusted Web Activity must meet Lighthouse's PWA install quality
-criteria and additional Android-specific criteria such as [policy compliance][9].
+Android app.
+
+Web content in a Trusted Web Activity must meet the PWA installability criteria.
+
+Additionally, in order to match the behaviour users expect from Android applications,
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md:73]:
```suggestion
Additionally, to match the behavior users expect from Android applications,
```

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 2]
```diff
@@ -66,8 +66,23 @@ nightly version.
...
+[Chrome 86 introduced a change][18] where failing to handle the following scenarios is considered a
+crash:
+
+  - Failure to verify digital asset links at application launch.
+  - Failure to return HTTP 200 for an offline network resource request.
+  - An HTTP 404 or 5xx error in the application.
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md:79]:
```suggestion
  - An application resource returning an HTTP 404 or 5xx error.
```

Comment (jpmedley) [reply src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md:79]: I feel like I need some extra context here. If I didn't correctly infer what's returning 404 and 5xx, come up with your own wording.

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 3]
```diff
@@ -66,8 +66,23 @@ nightly version.
...
+
+When one of those scenarios happen in the Trusted Web Activity, it will cause a user visible
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md:81]:
```suggestion
When one of those scenarios happens in the Trusted Web Activity, it causes a user visible
```

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 4]
```diff
@@ -66,8 +66,23 @@ nightly version.
...
+crash of the Android application. Check out the [guidance][19] on handling those scenarios in your
+Service Worker.
```
Comment (jpmedley) [inline src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md:83]:
```suggestion
service worker.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md (hunk 2 above)
Comment (andreban) [reply src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md:79]: How does "A navigation request returning an HTTP 404 or 5xx error" sound?

Comment (jpmedley) [reply src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md:79]: Sounds great.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md [hunk 5]
```diff
@@ -66,8 +66,23 @@ nightly version.
...
+
+Web content in a Trusted Web Activity must meet the PWA installability criteria.
+
+Additionally, to match the behavior users expect from Android applications,
+[Chrome 86 introduced a change][18] where failing to handle the following scenarios is considered a
+crash:
+
+  - Failure to verify digital asset links at application launch.
+  - Failure to return HTTP 200 for an offline network resource request.
+  - An HTTP 404 or 5xx error in the application.
```
Comment (andreban) [inline src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md:79]:
```suggestion
  - A navigation request returning an HTTP 404 or 5xx error".
```

File: src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md (hunk 2 above)
Comment (andreban) [reply src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md]: updated!

**Review by jpmedley — APPROVED** [review]

**Comment by kaycebasques** [comment]
Test comment for my reviewbot (everyone else can ignore)

**Pull request merged**